packages/shared/lib/integrations/scripts/connection/github-app-oauth-post-connection.ts

```diff
@@ -0,0 +1,17 @@
+import type { InternalNango as Nango } from './connection.manager.js';
+
+export default async function execute(nango: Nango) {
+    const response = await nango.proxy({
+        endpoint: `/user`
+    });
+
+    if (!response || !response.data) {
+        return;
+    }
+
+    const handle = response.data.login;
+
+    const connection = await nango.getConnection();
+
+    await nango.updateConnectionConfig({ handle, userCredentials: connection.credentials });
```

Comment:
Move the credentials to the connectionConfig because once the request is approved it will overwrite the credentials. So just in case our customer wants to use github user the credentials we keep them around.